### DISCUSSION TITLE
List all relatives links that point to a given page

### DISCUSSION THREAD

**Discussion by LunaticMuch** [facebook/docusaurus#7888] [Q&A]
Does the `router` allow to list all the relative links that are parent of a page?
Example:
- Page A has a link to Page B and Page D
- Page B has a link to page D
- Page X has a link to Page D

when rendering page D, I want to the a list parents, ie: Page A, Page B and Page X.

**Comment by slorber** [comment]
ATM we don't support bi-directional linking. We'll look into it in the future.

Related to https://github.com/facebook/docusaurus/issues/7618
